report/outbreak_no_intervention.qmd

```diff
@@ -0,0 +1,136 @@
+---
+title: H5N1 outbreak scenarios
+---
+
+This report requires two R packages: {epichains} and {ggplot2}, the former to simulate the branching process model and the latter to visualise the results.
```

Comment:
I'm not sure we need this here. The scientific report should not have any code or code-related references. We could add it under a section on "code and data availability" or something of the sort.

```suggestion

```